src/legacy/js/functions/_t4ArticleEditor.js

```diff
@@ -76,25 +76,20 @@ function articleEditor(collectionId, data) {
     data.description.metaDescription = $(this).val();
   });
 
-  /* The checked attribute is a boolean attribute, which means the corresponding property is true if the attribute
-   is present at all—even if, for example, the attribute has no value or is set to empty string value or even "false" */
-  var checkBoxStatus = function () {
-    if (data.description.nationalStatistic === "false" || data.description.nationalStatistic === false) {
-      return false;
-    } else {
-      return true;
-    }
-  };
+  $("#natStat-checkbox").click(function () {
+      data.description.nationalStatistic = $("#natStat-checkbox").prop('checked') ? true : false;
+  });
 
-  $("#metadata-list input[type='checkbox']").prop('checked', checkBoxStatus).click(function () {
-    data.description.nationalStatistic = $("#metadata-list input[type='checkbox']").prop('checked') ? true : false;
+  $("#articleType-checkbox").click(function () {
+      data.isPrototypeArticle = $("#articleType-checkbox").prop('checked') ? true : false;
```

Comment:
I don't think we need this to be a ternary operator, I think `.prop('checked')` should return truthy or falsey. You may want to keep it if you find it more readable though.